CHANGES.rst

```diff
@@ -3,6 +3,10 @@ Changes
 
 - remove reference to vendored library ``django.utils.six`` in favor of ``six`` (gh-526)
 
+2.7.1 (2019-03-20)
```

Comment:
The should be `Unreleased` until we release.

Reply:
Sorry I missed that! Last fix and then I'll approve lol

Reply:
Done